test/jdk/java/foreign/enablenativeaccess/TestEnableNativeAccess.java

```diff
@@ -30,7 +30,7 @@
  * @library /test/lib
  * @build TestEnableNativeAccess
  *        panama_module/*
- *        org.openjdk.foreigntest.PanamaMainUnnamedModule
+ *        org.openjdk.foreigntest.unnamed.PanamaMainUnnamedModule
```

Comment:
Had to move these classes to a different package, since when we put the same package on the class and module path, the package in the module hides the one on the class path.